Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN --mount=type=cache,sharing=private,target=/var/lib/apt/lists \
   build-essential git libpq-dev libyaml-dev unzip
 
 # Download skedjewel binary.
-ARG SKEDJEWEL_VERSION=v0.0.15
+ARG SKEDJEWEL_VERSION=v0.0.16
```

Comment:
https://github.com/davidrunger/skedjewel/releases/tag/v0.0.16

We need this version bump to schedule a PgHero Sidekiq job every 5 minutes (`PgHero::CaptureQueryStats: '**:%5'`).